README.md

```diff
@@ -27,6 +27,8 @@ Then, either make sure that `${GOPATH}/bin` is in your PATH, or explicitly add t
 "appraise" git alias by running the following command.
 
     git config --global alias.appraise '!'"${GOPATH}/bin/git-appraise"
+    # Windows:
```

Comment:
I think this needs to be three lines like:

```

#Windows

```

... and then the subtitle will be formatted the way you want.

Reply:
I actually meant to make that display like a Bash comment not a MarkDown heading but I'm happy to change it to that!